corehq/motech/repeaters/tests/test_proxy_models.py

```diff
@@ -183,3 +186,31 @@ def test_repeaters_are_synced_to_couch(self):
             {Repeater.wrap(r).repeater_type for r in repeaters},
             {r.repeater_type for r in self.all_repeaters}
         )
+
+
+class ModelAttrEqualityHelper(SimpleTestCase):
```

Comment:
@orangejenny Would like to have your inputs in the helper. Do you think it would be worth adding it in the couch-to-sql documentation and using it in future migrations?

Reply:
I like this, yeah, seems like a good check. I also like the comments you have in `_extra_attrs_in_couch_model` noting which attributes were removed, etc. - that seems like a useful reference for reviewers.

Do you think it'd be feasible to add the generation of these tests to the other couch to sql code generation? Then the author would just fill in the definitions for `_extra_attrs_in_couch_model`/`_new_attrs_in_sql`.

Reply:
Thanks, @orangejenny for the suggestion. Generating these tests automatically seems to be a great idea.
I would do it as a part of another PR which would add the documentation too.

Reply:
stellar :)